locations/spiders/soul_pattinson_chemist_au.py

```diff
@@ -0,0 +1,21 @@
+from scrapy import Selector, Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class SoulPattinsonChemistAUSpider(Spider):
+    name = "soul_pattinson_chemist_au"
+    item_attributes = {"brand": "Soul Pattinson Chemist", "brand_wikidata": "Q117225301"}
+    start_urls = ["https://soulpattinson.com.au/wp-admin/admin-ajax.php?action=store_search&autoload=1"]
+
+    def parse(self, response):
+        for location in response.json():
+            item = DictParser.parse(location)
+            item["name"] = location["store"].replace("&#038;", "&")
```

Comment:
Is `html.unescape` a better solution?

```suggestion
            item["name"] = html.unescape(location["store"])
```

Reply:
Thanks. It was just the ampersand symbol that is escaped but I've switched over to use the more generic html.unescape method.